vc.adoc

```diff
@@ -1076,6 +1087,163 @@ Content-Type: application/json;charset=UTF-8
 } ]
 ----
 
+[[group_exp]]
+== link:#group_exp_api[GroupExp Pro API Guide]
+
+[#author .author]#Ihor Karpylenko# +
+
+[[group_exp_api]]
+=== link:#group_exp_api[Fetch Categories based on GroupExp data]
+
+`POST /api/groupexp/v1.0/content-providers/{provider}/get-category-asset`
+
+This operation returns all category assets based on GroupExp data.
+
+[[_request_headers_group_exp]]
+==== link:#_request_headers_group_exp[Request headers]
+
+[width="100%",cols="50%,50%",options="header",]
+|===
+|Name |Value
+|Accept |application/json
+|Content-Type |application/json
+|authorization |apiKey xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+|===
+
+[[_path_parameters_group_exp]]
+==== link:#_path_parameters_group_exp[Path parameters]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Parameter |Type |Optional |Description
+|provider |String |false |Unique ID of the Content Provider.
+|===
+
+[[_query_parameters_group_exp]]
+==== link:#_query_parameters_group_exp[Query parameters]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Parameter |Type |Optional |Description
+|exerciserUuid |String |false |Exerciser Unique ID.
+|===
+
+[[_request_fields_group_exp]]
+==== link:#_request_fields_group_exp[Request fields]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Parameter |Type |Optional |Description
+|body |String |false |GroupExp objects in JSON format.
+|===
+Example:
+----
+{
+   "provider":"GroupExp v1",
```

Comment:
I would like to rename it somehow. For example, `source`

Reply:
fixed